### ISSUE TITLE
meteor version

### ISSUE THREAD

**Issue by dcsan** [scottpham/twitterBot#4]
is this dependent on any python NLP apis? 
have you seen http://superscriptjs.com ?

be interesting to combine these.

**Comment by scottpham** [comment]
No, there's no NLP apis involved here. The idea was that a front-end like a google spreadsheet would populate the tweets. I've not seen superscriptjs.com but it looks interesting!
